flink/README.md

```diff
@@ -150,13 +150,16 @@ Received a total of 1 row
 
 See [Real-time wide table](../docs/real-time_wide_table.md)
 
+## Partition table supports
+
+See [flink_partition_table](../docs/flink_partition_table.md)
+
 ## Supports and Limitations
 
 TiBigDate/Flink supports writing data to and reading from clustered index tables, which is a new feature in TiDB-5.0.0.
 
 TiBigDate/Flink does not support the following features: 
 - bypass-write tables with auto random column
```

Comment:
Maybe delete this too
https://github.com/tidb-incubator/TiBigData/pull/234

Reply:
OK, I have deleted this.